crates/flox-rust-sdk/src/models/environment/mod.rs

```diff
@@ -604,6 +630,14 @@ mod test {
         "version": 1
     }"#;
 
+    const MANAGED_ENV_POINTER: Lazy<EnvironmentPointer> = Lazy::new(|| {
+        EnvironmentPointer::Managed(ManagedPointer {
```

Comment:
suggestion: `Lazy` and other `Cell` based globals should not be const.
const will basically put copies of this (slighly overheady) object everywhere its being used which isnt the point of using a lazy value to begin with.
`static` will use the same pointer

```suggestion
    static MANAGED_ENV_POINTER: Lazy<EnvironmentPointer> = Lazy::new(|| {
```